Add parameters and tip notes (#24640)
newrelic/docs-website: src/content/docs/opentelemetry/database/mssql/windows-hosted.mdx

<CollapserGroup>
  <Collapser
    id="db-only"

>     title="Core Observability"

Reviewer: ```suggestion
    id="standard"
    title="Standard configuration"
```

**Enter your values below and see the configuration update in real-time:**

<AgentConfig
  inputOptions={[
    {
      name: 'username',
      label: 'SQL Server Username',
      codeLine: 8,
      defaultValue: 'newrelic',
      toolTip: 'Enter your SQL Server username',
    },
    {
      name: 'password',
      label: 'SQL Server Password',
      codeLine: 9,
      defaultValue: 'your_password',
      toolTip: 'Enter your SQL Server password',
    },
    {
      name: 'server',
      label: 'Server Hostname/IP',
      codeLine: 10,
      defaultValue: 'sqlserver.example.com',
      toolTip: 'Enter your SQL Server hostname or IP address',
    },
    {
      name: 'port',
      label: 'Server Port',
      codeLine: 11,
      defaultValue: '1433',
      toolTip: 'Enter your SQL Server port number (default: 1433)',
    },
    {
      name: 'endpoint',
      label: 'New Relic OTLP Endpoint',
      codeLine: 50,
      defaultValue: 'https://otlp.nr-data.net',

>       toolTip: 'Enter your New Relic OTLP endpoint (e.g., https://otlp.nr-data.net or https://otlp.eu01.nr-data.net). For more information, refer to /docs/opentelemetry/best-practices/opentelemetry-otlp/',

Reviewer: ```suggestion
      toolTip: 'Enter your New Relic OTLP endpoint. For more information, refer to https://docs.newrelic.com/docs/opentelemetry/best-practices/opentelemetry-otlp/',
```

    metrics:
      level: none

  pipelines:
    metrics:
      receivers: [nrsqlserver]
      processors: [memory_limiter, batch]
      exporters: [otlphttp]

    logs:
      receivers: [nrsqlserver]
      processors: [memory_limiter, batch]
      exporters: [otlphttp]`}
  fileName="mssql-config.yaml"
  tipMdx={{ body: null }}
  onChange={() => {}}
/>

<Callout variant="tip">

Use the form above to enter your specific values, and the configuration will update in real-time. Copy this configuration to `C:\Program Files\NRDOT Collector Host\mssql-config.yaml` for database-only monitoring

</Callout>

  </Collapser>

  <Collapser
    id="full-config-sql"

>     title="All Capabilities"

Reviewer: ```suggestion
    id="full"
    title="Full-feature configuration"
```

</Callout> 

<AgentConfig
  inputOptions={[
    {
      name: 'username',
      label: 'SQL Server Username',
      codeLine: 4,
      defaultValue: 'newrelic',
      toolTip: 'Enter your SQL Server username for authentication',
    },
    {
      name: 'password',
      label: 'SQL Server Password',
      codeLine: 5,
      defaultValue: 'your_password',
      toolTip: 'Enter your SQL Server password for authentication',
    },
    {
      name: 'server',
      label: 'Server Hostname/IP',
      codeLine: 6,
      defaultValue: 'sqlserver.example.com',
      toolTip: 'Enter your SQL Server hostname or IP address',
    },
    {
      name: 'port',
      label: 'Server Port',
      codeLine: 7,
      defaultValue: '1433',
      toolTip: 'Enter your SQL Server port number (default: 1433)',
    },
    {
      name: 'endpoint',
      label: 'New Relic OTLP Endpoint',
      codeLine: 177,
      defaultValue: 'https://otlp.nr-data.net:4318',

>       toolTip: 'Enter your New Relic OTLP endpoint (e.g., https://otlp.nr-data.net:4318). For more information, refer to /docs/opentelemetry/best-practices/opentelemetry-otlp/',

Reviewer: ```suggestion
      toolTip: 'Enter your New Relic OTLP endpoint. For more information, refer to https://docs.newrelic.com/docs/opentelemetry/best-practices/opentelemetry-otlp/',
```

<CollapserGroup>

  <Collapser
    id="domain-database-only"

>     title="Core Observability"

Reviewer: ```suggestion
    id="standards"
    title="Standard configuration"
```

      insecure: false
    compression: gzip

service:
  telemetry:
    metrics:
      level: none

  pipelines:
    metrics:
      receivers: [nrsqlserver]
      processors: [memory_limiter, batch]
      exporters: [otlphttp]

    logs:
      receivers: [nrsqlserver]
      processors: [memory_limiter, batch]
      exporters: [otlphttp]
```

<Callout variant="important">
Replace the following values:

- `<HOSTNAME-OR-RDS-ENDPOINT>`: Your server hostname (on-premises) or RDS endpoint

> - `<YOUR_NEWRELIC_OTLP_ENDPOINT>`: Your New Relic OTLP endpoint (e.g., https://otlp.nr-data.net). For more information, refer to [New Relic OTLP endpoints documentation](/docs/opentelemetry/best-practices/opentelemetry-otlp/)

Reviewer: ```suggestion
- `<YOUR_NEWRELIC_OTLP_ENDPOINT>`: Your New Relic OTLP endpoint. For more information, refer to [New Relic OTLP endpoints documentation](/docs/opentelemetry/best-practices/opentelemetry-otlp/).
```

- `<YOUR_NEWRELIC_LICENSE_KEY>`: Your New Relic license key
</Callout>

  </Collapser>

  <Collapser
    id="domain-host-database"

>     title="All Capabilities"

Reviewer: ```suggestion
    id="ffc"
    title="Full-feature configuration"
```

Replace the following values:

- `<HOSTNAME-OR-RDS-ENDPOINT>`: Your server hostname (on-premises) or RDS endpoint

> - `<YOUR_NEWRELIC_OTLP_ENDPOINT>`: Your New Relic OTLP endpoint (e.g., https://otlp.nr-data.net). For more information, refer to [New Relic OTLP endpoints documentation](/docs/opentelemetry/best-practices/opentelemetry-otlp/)

Reviewer: ```suggestion
- `<YOUR_NEWRELIC_OTLP_ENDPOINT>`: Your New Relic OTLP endpoint. For more information, refer to [New Relic OTLP endpoints documentation](/docs/opentelemetry/best-practices/opentelemetry-otlp/).
```

<CollapserGroup>

  <Collapser
    id="gmsa-database-only"

>     title="Core Observability"

Reviewer: ```suggestion
    id="gmsa-standard"
    title="Standard configuration"
```

      insecure: false
    compression: gzip

service:
  telemetry:
    metrics:
      level: none

  pipelines:
    metrics:
      receivers: [nrsqlserver]
      processors: [memory_limiter, batch]
      exporters: [otlphttp]

    logs:
      receivers: [nrsqlserver]
      processors: [memory_limiter, batch]
      exporters: [otlphttp]
```

<Callout variant="important">
Replace the following values:

- `<HOSTNAME-OR-RDS-ENDPOINT>`: Your server hostname (on-premises) or RDS endpoint

> - `<YOUR_NEWRELIC_OTLP_ENDPOINT>`: Your New Relic OTLP endpoint (e.g., https://otlp.nr-data.net). For more information, refer to [New Relic OTLP endpoints documentation](/docs/opentelemetry/best-practices/opentelemetry-otlp/)

Reviewer: ```suggestion
- `<YOUR_NEWRELIC_OTLP_ENDPOINT>`: Your New Relic OTLP endpoint). For more information, refer to [New Relic OTLP endpoints documentation](/docs/opentelemetry/best-practices/opentelemetry-otlp/).
```

- `<YOUR_NEWRELIC_LICENSE_KEY>`: Your New Relic license key
</Callout>

  </Collapser>

  <Collapser
    id="gmsa-host-database"

>     title="All Capabilities"

Reviewer: ```suggestion
    id="gmsa-ffc"
    title="Full feature configuration"
```

Replace the following values:

- `<HOSTNAME-OR-RDS-ENDPOINT>`: Your server hostname (on-premises) or RDS endpoint

> - `<YOUR_NEWRELIC_OTLP_ENDPOINT>`: Your New Relic OTLP endpoint (e.g., https://otlp.nr-data.net). For more information, refer to [New Relic OTLP endpoints documentation](/docs/opentelemetry/best-practices/opentelemetry-otlp/)

Reviewer: ```suggestion
- `<YOUR_NEWRELIC_OTLP_ENDPOINT>`: Your New Relic OTLP endpoint. For more information, refer to [New Relic OTLP endpoints documentation](/docs/opentelemetry/best-practices/opentelemetry-otlp/).
```
